src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -65,8 +65,8 @@ private static AddressVo getCreateAddressResponseData(InquiryAddress inquiryAddr
     }
 
     private static List<EducationVo> getCreateEducationResponseData(List<InquiryEducation> inquiryEducation, List<EducationVo> educationVos) {
-        EducationVo educationVo=new EducationVo();
         for(InquiryEducation inquiryEducation1:inquiryEducation){
+            EducationVo educationVo=new EducationVo();
             educationVo.setId(inquiryEducation1.getId());
             educationVo.setAggregateMarks(inquiryEducation1.getAggregateMarks() == null ? null : inquiryEducation1.getAggregateMarks());
```

Comment:
If we are setting null in case of null, this check can be avoided.

Reply:
okay resolved in commit 6ffb6af